docs/e-Signet OIDC.postman_collection.json

```diff
@@ -1,57 +1,48 @@
 {
 	"info": {
-		"_postman_id": "496855d4-693e-43f6-b0be-dd22c2cd3fa1",
```

Comment:
in the file names in docs for e-Signet, for lets follow any one way consistently, e-Signet or eSignet everywhere

Reply:
Done